extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -240,7 +240,7 @@ function arrayprintObj(  &$parser, $frame, $args ) {
 * usage
 *   {{#arrayindex:arrayid|index}}
 */
-    function arrayindex( &$parser, $arrayid , $index , $options='') {
+    function arrayindex( $parser, $arrayid , $index , $options='') {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)